.travis.yml

```diff
@@ -3,19 +3,6 @@ notifications:
   email: false
 jobs:
   include:
-    - stage: test
-      node_js: 6
-      script: npm run test
-    - node_js: 8
-      script: npm run test
-    - node_js: 10
-      script: npm run test
-    - node_js: 12
-      script:
-        - npm run build
-        - npm run lint:commit-travis
-        - npm run lint
-        - npm run test
```

Comment:
Let's keep Travis for a while to have a comparison?

Reply:
sure, no problem :+1: